Microsoft.Identity.Web/ClaimsPrincipalExtensions.cs

```diff
@@ -21,16 +21,35 @@ public static class ClaimsPrincipalExtensions
         public static string GetMsalAccountId(this ClaimsPrincipal claimsPrincipal)
         {
             string userObjectId = claimsPrincipal.GetObjectId();
+            string nameIdentifierId = claimsPrincipal.GetNameIdentifierId();
             string tenantId = claimsPrincipal.GetTenantId();
+            string policyId = claimsPrincipal.GetPolicyId();
```

Comment:
>GetPolicyId [](start = 46, length = 11)

is this a b2c policy or the concept of a policy for aad? we've been having discussions on naming the b2c policies differently, such as userFlow, to not confuse w/AAD policies, but this has not been determined, as we have docs all over which use both.

Reply:
@jennyf19 this method is getting whatever is in the `tfp` claims. Personally I dont know if this is a B2C only claim. @kalyankrishna1 @jmprieur is `tfp` claims B2C only?

Reply:
@TiagoBrenck yes...`tfp` claims is unique to B2C